ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -371,12 +377,23 @@ def add_capacity(osd_size_capacity_requested):
     storageDeviceSets->count = (capacity reqested / osd capacity ) + existing count storageDeviceSets
 
     """
-
-    sc = get_storage_cluster()
-    old_storage_devices_sets_count = get_deviceset_count()
     osd_size_existing = get_osd_size()
-    new_storage_devices_sets_count = int((osd_size_capacity_requested / osd_size_existing)
-                                         + old_storage_devices_sets_count)
+    device_sets_required = int(osd_size_capacity_requested / osd_size_existing)
+    old_storage_devices_sets_count = get_deviceset_count()
+    new_storage_devices_sets_count = int(device_sets_required + old_storage_devices_sets_count)
+    lvpresent = check_local_volume()
+    if lvpresent:
+        final_device_list = get_new_device_paths(device_sets_required, osd_size_capacity_requested)
+        param = f"""[{{ "op": "replace", "path": "/spec/storageClassDevices/0/devicePaths",
+                                                 "value": {final_device_list}}}]"""
+        lvcr = get_local_volume_cr()
+        lvcr.patch(
+            resource_name=lvcr.get()['items'][0]['metadata']['name'],
+            params=param.strip('\n'),
+            format_type='json'
+        )
+        check_pvs_created(int(len(final_device_list) / new_storage_devices_sets_count))
+    sc = get_storage_cluster()
```

Comment:
This is not getting used anywhere

Reply:
It is, For patching the storage cluster.

Reply:
I'm not seeing where `sc` is used either.

Reply:
I rearranged the code a bit for better flow. 
It is being used to patch the storage cluster as mentioned above.
You can check here:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/storage_cluster.py#L383

Reply:
@clacroix12 ^^^^